manim/mobject/graphing/coordinate_systems.py

```diff
@@ -144,7 +145,7 @@ def __init__(
         self.y_length = y_length
         self.num_sampled_graph_points_per_tick = 10
 
-    def coords_to_point(self, *coords: Sequence[ManimFloat]):
+    def coords_to_point(self, *coords: Point3D):
```

Comment:
```suggestion
    def coords_to_point(self, *coords: ManimFloat):
```

This is what got me confused regarding my other comment